### PR TITLE
Add example code into README.md

### PR DESCRIPTION
**Pull request by akira345**
Hi.
The sample code in the Readme does not appear to have any examples that include inherited members.
Added example using hidden option ``` includeInherited ```.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,6 +401,11 @@ const api = new Gitlab({
 // Listing users
 let users = await api.Users.all();
 
+// Listing project members(including inherited members)
+// see. https://docs.gitlab.com/ee/api/members.html#list-all-members-of-a-group-or-project-including-inherited-members
+const projectId = 9999;
```
Comment (Jetersen) [inline README.md]: does not have to be a projectId could just as well be the group/project ;)

Comment (akira345) [reply README.md]: Sorry , I don't understand well ... 

Should I add sample code that uses ```GroupMembers```? or elase magic number ```projectId```?

ex.
```
// Or using Promise-Then notation
api.Projects.all().then((projects) => {
  console.log(projects);
  // add sample codes.
});
```

Comment (Jetersen) [reply README.md]: For the example I do not think project id is helpful.
Yes you can use project id if you know them by heard or doing a sequence of nested api calls like first getting projects and than using those project to.

My suggestion for the example is to the namespace path so `group/project` or `gitlab-org/gitlab` just like in the url to the project.
https://gitlab.com/gitlab-org/gitlab

Comment (akira345) [reply README.md]: Thank you for pointing this out.
Updated sample code.

**Pull request merged**